### ISSUE TITLE
[🕹️] Allow uploading of doc from visitor

### ISSUE THREAD

**Issue by ykburda** [mfts/papermark#647]
### What side quest or challenge are you solving?

Feature request

### Points

?

### Description

Sometime we need to allow visitors to upload eg: some specific docs to dataroom. How can it be possible? There is no way to do this now. We need to see uploaded documents by visitors. Of course they will not see each others docs. Only admin will see this docs by visitor. 

### Provide proof that you've completed the task

?

**Comment by mfts** [comment]
Absolutely! That's a great suggestion and fully aligned with our roadmap.

**Comment by AshishViradiya153** [comment]
Any updates @PraveenThanikachalam? May I look into this one?

**Comment by AshishViradiya153** [comment]
Any updates @PraveenThanikachalam?

**Comment by birajkashyap** [comment]
I have worked on full stack applications like social media app, where uploading of picture was a major deal. I can handle uploading doc files too. So please consider assigning this issue to me.

**Comment by RajuGangitla** [comment]
@PraveenThanikachalam are you still working on this one is there any update ?

**Comment by RajuGangitla** [comment]
@mfts ??

**Comment by mfts** [comment]
@RajuGangitla i think @AshishViradiya153 is already working on something here.

**Comment by RajuGangitla** [comment]
@mfts ok
 but it's assigned to some other person 🤔

**Comment by AshishViradiya153** [comment]
@mfts, I may finish today or maybe tomorrow and will be submit a PR.

**Comment by AshishViradiya153** [comment]
https://github.com/user-attachments/assets/4899917a-79db-430a-b028-1f33b13d63cf

**Comment by AshishViradiya153** [comment]
![image](https://github.com/user-attachments/assets/24215ec7-89d7-41fb-a4ef-105983c6ab31)

**Comment by AshishViradiya153** [comment]
@mfts is there anything extra to add.

**Comment by DeepaPrasanna** [comment]
Hey @AshishViradiya153 , are you still working on this?
